doc/source/administrator/authentication.md

```diff
@@ -0,0 +1,427 @@
+(authentication)=
+
+# Authentication and authorization
+
+Authentication is about identity, while _authorization_ is about access rights.
+In this section you will learn how to configure both. For example, you can
+configure to authenticate GitHub users and authorize those part of a specific
+GitHub organization.
+
+Before configuring this, you should have [setup HTTPS](https).
+
+## Useful understanding
+
+### Authenticator classes
+
+As JupyterHub can't know how to authenticate all kinds of users, it expose
+configuration to extend its capabilities to rely on a dedicated [_authenticator
+class_](https://jupyterhub.readthedocs.io/en/stable/reference/authenticators.html).
+Several such classes are already made available through [installed Python
+packages](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/master/images/hub/requirements.txt).
```

Comment:
Is there any magic in MyST that would allow the tag to appear here instead of `master`? So that the stable docs corresponding to a release could link to the correct set of packages, whereas the dev/master docs could link to `master`?

Reply:
This is now resolved through https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/1968/files#diff-d8d3ed25802824d15bf411f8e97416d8fc6f9247e821b0617f2b869dc584b99cR102-R109.